dev-reqs.txt

```diff
@@ -12,5 +13,26 @@ zope.component == 3.11
 zope.deprecation == 3.5
 zope.event == 3.5.1
 zope.interface == 3.8
+#
+# BIG HUGE WARNING
+#
+# I don't know how but mozsvc 0.8 is causing WEIRD behaviour in M2Crypto.  If
+# you see errors like this:
+#
+# TypeError: in method 'bio_new_file', argument 1 of type 'char const *'
+#
+# It's something in the middle.  I'm still not sure what.  0.3 works so we're
+# sticking with it for the time being for reasons of expediency.
+#
+# rtilder  Thu Dec 11 11:29:03 PST 2014
+#
+#mozsvc == 0.8
```

Comment:
You shouldn't need to copy all these reqs from prod. You can include them with [-r prod-reqs.txt](https://pip.pypa.io/en/latest/reference/pip_install.html#cmdoption-r). If you need different modules on different servers you could do something like ``-r shared-reqs.txt``.